### ISSUE TITLE
net/haproxy: change server state on-the-fly

### ISSUE THREAD

**Issue by fraenki** [opnsense/plugins#486]
split from #165:

And another thing in this context is the possibility to set a server in drain mode.
Existing connections keeps on their backend server. But new connections goes to "ready" server only.
http://cbonte.github.io/haproxy-dconv/1.8/management.html#9.3-set%20server

We should probably start by supporting the following features:

```
set server <backend>/<server> state [ ready | drain | maint ]
set server <backend>/<server> weight <weight>[%]
```

**Comment by fraenki** [comment]
I think this feature would be even more useful when it would allow to send any available command to any server. For this to work we probably need a new page, similar to the stats page, that shows all servers and their current/live state. A new button would allow to send commands to a server. Additionally it might be useful to be able to send a command to multiple servers at once.

This probably obsoletes the need to reload/restart HAProxy in many situations.

**Comment by fraenki** [comment]
implemented in #2213
